modules/configuration/src/test/java/org/apache/ignite/internal/configuration/notifications/ConfigurationListenerTest.java

```diff
@@ -915,4 +957,17 @@ void testGetConfigFromNotificationEventOnUpdate() throws Exception {
         
         configuration.elements().get(key).str().update(newVal).get(1, SECONDS);
     }
+
+    @Test
+    void listenerGetsNotifiedOnCommonPropertyChangeInPolymorphicBaseConfigUnderNoNamedConfig() throws Exception {
+        AtomicInteger intHolder = new AtomicInteger();
```

Comment:
Add new line below

Reply:
Why?

Reply:
Hard to read

Reply:
Oh, I see why, it gets used later in the method, not just immediately.

Added the line.